contracts/ERC721ControlledFactory.sol

```diff
@@ -44,6 +48,7 @@ contract ERC721ControlledFactory {
     ERC721Controlled result = ERC721Controlled(payable(Create2.deploy(0, _salt(msg.sender), erc721ControlledBytecode)));
     nonces[msg.sender] = nonces[msg.sender].add(1);
     result.initialize(name, symbol, baseURI, msg.sender);
+    emit ERC721ControlledCreated(address(result));
```

Comment:
Let's make sure to add an expectation for this event in the test
`
await expect(factory.createERC721Controlled(....)).to.emit(factory, 'ERC721ControlledCreated')
`